test/acceptance/cli-test.js

```diff
@@ -725,14 +725,14 @@ describe('ember-template-lint executable', function () {
           shell: true,
         });
 
-        expect(result.exitCode).toEqual(1);
+        expect(result.exitCode).toEqual(0);
```

Comment:
Why should this be 0?

We enabled a lint, And the template violates that lint, ya?

Reply:
I reverted commits and found a different solution.